playbooks/rds_backup_monitoring_setup.yml

```diff
@@ -0,0 +1,168 @@
+---
+- name: Prepare controller variables
+  import_playbook: setup_scenarios_controller.yml
+
+- name: Prepare variables
+  hosts: localhost
+  environment:
+    OS_CLIENT_CONFIG_FILE: "{{ clouds_yaml_file }}"
+    TF_VAR_network_id: "{{ main_network_id }}"
+    TF_VAR_router_id: "{{ main_router_id }}"
+    TF_VAR_subnet_id: "{{ main_subnet_id }}"
+    TF_VAR_controller_ip: "{{ controller_public_ip }}"
+  vars:
+    scenario_name: rds_backup_monitoring
+    terraform_base_dir: terraform
+    requirements: requirements.txt
+    terraform_workspace: prod
```

Comment:
`terraform_workspace` is defined in the inventory, please do not rewrite it here